src/JSInterop/Microsoft.JSInterop/ref/Microsoft.JSInterop.netcoreapp3.0.cs

```diff
@@ -5,19 +5,19 @@ namespace Microsoft.JSInterop
 {
     public static partial class DotNetDispatcher
```

Comment:
Are we going to hide this in another namespace?

Reply:
I think we decided to put a pin on it and didn't get around to figuring it out. Does `Microsoft.JSInterop.Infrastructure` work? It's not directly relevant to user code, so that seems appropriate to me.